src/test/java/com/faforever/api/user/UserServiceTest.java

```diff
@@ -95,6 +96,8 @@
   private GlobalRatingRepository globalRatingRepository;
   @Mock
   private Ladder1v1RatingRepository ladder1v1RatingRepository;
+  @Mock
+  private ApplicationEventPublisher eventPublisher;
```

Comment:
no assert on event publishing and its content?

Reply:
Fixed now :)